datarobot_batch_scoring/detect.py

```diff
@@ -0,0 +1,91 @@
+import math
+import re
+
+
+class Detector(object):
```

Comment:
I mean, you have to thoroughly read the code, comments/docs string made for common-sense obvious information that gives heads up before code readeing.